tests/integration/test_github_client.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Tests for the github_client module."""
+
+import typing
+
+import pytest
+
+from repo_policy_compliance import target_branch_protection
+from repo_policy_compliance.exceptions import GithubClientError, InputError
+from repo_policy_compliance.github_client import GITHUB_TOKEN_ENV_NAME
+
+from .. import assert_
+
+
+@pytest.mark.parametrize(
+    "github_token_value, exception_context, exception_message",
+    [
+        pytest.param("", pytest.raises(InputError), "was not provided", id="github_token empty"),
+        pytest.param(
+            None, pytest.raises(GithubClientError), "Bad Credential error", id="github_token none"
+        ),
+        pytest.param(
+            "abcd",
+            pytest.raises(GithubClientError),
+            "Bad Credential error",
+            id="github_token wrong",
+        ),
+    ],
+)
+def test_github_token(
+    github_token_value: str | None,
+    exception_context: typing.ContextManager,
+    exception_message: str,
+    github_repository_name: str,
+    monkeypatch: pytest.MonkeyPatch,
+):
+    """
+    arrange: A github repository name and a missing github token.
```

Comment:
What does the "missing github token" refers to here?
It seems the `@pytest.mark.parametrize` is injecting a incorrect github token in one of the cases.
Would "bad github token" be a better description?

Reply:
Good catch, should be fixed now